src/dlgConnectionProfiles.cpp

```diff
@@ -722,22 +722,23 @@ QString dlgConnectionProfiles::readProfileData(const QString& profile, const QSt
 
 QPair<bool, QString> dlgConnectionProfiles::writeProfileData(const QString& profile, const QString& item, const QString& what)
 {
-    auto f = mudlet::getMudletPath(mudlet::profileDataItemPath, profile, item);
-    QFile file(f);
+    QSaveFile file(mudlet::getMudletPath(mudlet::profileDataItemPath, profile, item));
     if (file.open(QIODevice::WriteOnly | QIODevice::Unbuffered)) {
         QDataStream ofs(&file);
         if (mudlet::scmRunTimeQtVersion >= QVersionNumber(5, 13, 0)) {
             ofs.setVersion(mudlet::scmQDataStreamFormat_5_12);
         }
         ofs << what;
-        file.close();
+        if (!file.commit()) {
+            qDebug() << "dlgConnectionProfiles::writeProfileData: error writing custom profile data: " << file.errorString();
```

Comment:
It isn't "*custom*" data - it is any data that is stored as a file in the root of the profile's part of the file-system. You might actually want to say what the `item` is so someone who can actually see the debug output knows what has been messed up...
```suggestion
            qDebug() << "dlgConnectionProfiles::writeProfileData: error writing profile data: " << file.errorString();
```

Reply:
Updated.